content/source/docs/enterprise/api/organizations.html.md

```diff
@@ -339,3 +339,52 @@ curl \
 ### Sample Response
 
 The response body will be empty if successful.
+
+## Show the Entitlement Set
+This endpoint shows the [entitlements](../users-teams-organizations/entitlements.html) for an organization.
```

Comment:
Can these also just be included with the org? We usually have an Available Related Resources that points to a separate page in those cases, so it might be better to do this one that way?